.gitlab/workflows.yml

```diff
@@ -1,13 +1,8 @@
 workflow:
```

Comment:
This is probably a dumb question, but should these `.gitlab` files be exclusive to the GitLab mirror?

Reply:
Not at all a dumb question :-) 
Yea, these `.gitlab` files (this `workflow.yml` in particular) are used exclusively by our downstream GitLab mirror. I'd assume you're asking why we don't just keep this directory only in GitLab. From what I can tell, this is the only practical way to trigger a GitLab pipeline from GitHub (via the periodic mirroring). For example, I don't think that GitLab would allow CI pipelines on a branch (say a mirrored branch) to be controlled by a `.yml` file from another branch (some internal branch having this file).

Reply:
So it sounds like we'd need some way of mirroring this repo **and** merging in these .yml files on the same branch in GitLab.

Reply:
Digging into this a bit further, I think it should be possible to move this `workflow.yml` file to the downstream GitLab repository via the [`include` configuration](https://docs.gitlab.com/ee/ci/yaml/index.html#include). 

This would still require a thin `.yml` file pointing to the full `.yml` file, e.g.,

```
include:
  - project: 'GitLab mirror'
    ref: branch-contains-workflow-yml                                      # Git branch
    file: 'workflow.yml'
```

I guess we'd need an RFC to explore this option to check if pushing `workflow.yml` downstream is something we want to do.